ArpHeader/AmKernel/AmGlobalsImpl.h

```diff
@@ -40,8 +40,8 @@
 class BClipboard;
 class AmFilterAddOn;
 
-typedef vector<AmViewFactory*>			fac_vec;
-typedef vector<BMessenger>				observer_vec;
+typedef std::vector<AmViewFactory*>			fac_vec;
+typedef std::vector<BMessenger>				observer_vec;
```

Comment:
You can do `using std::vector` to avoid this, I think.

Reply:
Yup, I know. But it was faster to just search and replace all instances of "vector<" with "std::vector<" and same for all other things in std namespace.